packages/vue-components/src/PageNavButton.vue

```diff
@@ -19,30 +16,16 @@ import $ from './utils/NodeList';
 export default {
   data() {
     return {
-      hasPageNav: false,
-      hasIdentifier: false,
+      portalName: undefined,
       show: false,
-      src: '',
     };
   },
   computed: {
     showPageNav() {
-      return this.show && this.hasPageNav;
+      return this.show && this.portalName;
     },
   },
   methods: {
-    getPageNavContent() {
-      const wrapper = document.createElement('div');
-      const pageNavTitle = document.getElementsByClassName('page-nav-title')[0];
-      const pageNavLinks = document.getElementById('mb-page-nav');
-      if (pageNavTitle) {
-        wrapper.appendChild(pageNavTitle.cloneNode(true));
-      }
-      if (pageNavLinks) {
-        wrapper.appendChild(pageNavLinks.cloneNode(true));
-      }
-      return wrapper;
-    },
```

Comment:
I believe this method is still needed so that we can still pull in the page nav if the `page-nav` identifier is not present? As discussed [here](https://github.com/MarkBind/markbind/pull/1445#issuecomment-764419350), we should pull in the identifier first if they exist and if not, we should find the page nav and then pull that in.

Alternatively, we can also change the requirement such that users will have to explicitly specify the page nav with the identifier if that makes it better?

Reply:
> I believe this method is still needed so that we can still pull in the page nav if the page-nav identifier is not present? As discussed here, we should pull in the identifier first if they exist and if not, we should find the page nav and then pull that in.

I standardised this to a portal implementation as well